### PR TITLE
feat : Added loader on login , signup , logout and add to playlist

### PR DESCRIPTION
**Pull request by rutvikpumak**

### CODE REVIEW

File: src/pages/Auth/Login.js [hunk 1]
```diff
@@ -86,7 +92,7 @@ export function Login() {
 
           <Link to="/signup">
             <div className="auth-secondary-btn text-center">
-              Create New Account
+              Create New Account{" "}
```
Comment (anuj-kumary) [inline src/pages/Auth/Login.js:95]: Remove these unwanted curly braces

**Review by Rohanmond — APPROVED** [review]
code is looking good and liked the way you used PrivateRoute

**Review by anuj-kumary — APPROVED** [review]
The UI is looking good. Nice work!!
Good to Merge

File: src/pages/Auth/Login.js (hunk 1 above)
Comment (rutvikpumak) [reply src/pages/Auth/Login.js:95]: Its intentional

**Pull request merged**